pkg/apis/version/version_validation_test.go

```diff
@@ -14,19 +14,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package version
+package version_test
```

Comment:
why are the changes in this file included in this commit? seems to me that it's an unrelated cleanup

Reply:
Split into a separate PR: https://github.com/tektoncd/pipeline/pull/5319

Reply:
happy to lgtm this when these changes in this file are removed from this commit

Reply:
my bad-- for some reason I forgot it was possible to have some tests in a "version" package and some in a "version_test" package so I just assumed this would wait for 5319 to merge. done.